config/config.go

```diff
@@ -274,13 +274,20 @@ func SubWithDefault(config *viper.Viper, key string) *viper.Viper {
 	return sub
 }
 
+var HiddenKeys = map[string]bool{
```

Comment:
Can we do it the other way. Maintain the list of display keys and only send that. It's safer

Reply:
agrre, done